mbed_build/_internal/config/source.py

```diff
@@ -79,6 +89,18 @@ def _filter_target_overrides(data: dict, allowed_labels: List[str]) -> dict:
     return flattened
 
 
+def _split_target_overrides_by_type(data: dict) -> Tuple[dict, dict]:
+    """Split target override data into config overrides and cumulative overrides."""
+    config_overrides = {}
+    cumulative_overrides = {}
+    for key, value in data.items():
+        if key in METADATA_OVERRIDE_KEYS:
```

Comment:
So MEATDATA_OVERRIDE_KEYS == full list of cumulative overrides?  Its not the most obvious of names 🤔

Reply:
omg METADATA 🤦

Reply:
Nice catch! https://github.com/ARMmbed/mbed-build/pull/29/commits/14017428b54fc513a0fdfc0b2636dd40dc0c89de